commandline/src/test/java/tech/pegasys/web3signer/commandline/CommandlineParserTest.java

```diff
@@ -136,6 +136,28 @@ void eth2SubcommandSlashingDatabaseUrlNotRequiredWhenSlashingDisabled() {
     assertThat(result).isZero();
   }
 
+  @Test
+  void missingToInExportShowsError() {
+    String cmdline = validBaseCommandOptions();
+    cmdline = cmdline + "eth2 export";
+    parser.registerSubCommands(new MockEth2SubCommand());
+    final int result = parser.parseCommandLine(cmdline.split(" "));
+
+    assertThat(result).isNotZero();
+    assertThat(commandError.toString()).contains("--to has not been specified");
+  }
+
```

Comment:
can  some tests be added to show that missing the db url for import & export results in an error

Reply:
done